plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -57,6 +57,12 @@ func ComputeTaintTolerationPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *
 	if node == nil {
 		return schedulerapi.HostPriority{}, fmt.Errorf("node not found")
 	}
+	// get taints from node info cache.
+	taints, err := nodeInfo.Taints()
+	if err != nil {
+		// fallback to computing taints from node Spec
+		taints = node.Spec.Taints
```

Comment:
i could be missing something, if so feel free to re-open, but my opinion is that for any non-obvious performance changes we need a measurement in place, and this change is non-obvious to me - it might be obvious to someone else however, if so, then feel free to reopen.